src/draw/renesas/dave2d/lv_draw_dave2d.h

```diff
@@ -27,6 +27,9 @@ extern "C" {
 #include "../../lv_draw_triangle.h"
 #include "../../lv_draw_buf.h"
 
+#include "../../lv_draw_image_private.h"
+#include "../../../misc/lv_area_private.h"
```

Comment:
Question, if those include files are intended to be private, they should not be exposed to the public interface file, right? What about adding these include only on file that call these private functions, I know it would involve more effort, but it will keep the design of the public interface consistent.